### ISSUE TITLE
Can you please make this into a chatbot (telegram/discord)?

### ISSUE THREAD

**Issue by damare01** [dteviot/WebToEpub#525]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
when a person send a novel's link to the bot the bot will download the novel and send the file to the user.

**Comment by damare01** [comment]
please

**Comment by gamebeaker** [comment]
@damare01
i think the answer is no see #487 
question: why would you like this or why do you need this?

**Comment by damare01** [comment]
so that we can host it on our own, but if it's troublesome nevermind.

**Comment by dteviot** [comment]
As this is the second time I've got a request like this, I'll give the reason for saying "no".

I assume that the request is to provide something like the Telegram mode of [Novel-Grabber by Flemish](https://github.com/Flameish/Novel-Grabber/blob/master/README.md#how-to-use)

The fundamental problem is WebToEpub is just not designed to do this.
WebToEpub is a browser extension.  That is, it's designed to live inside a web browser and takes advantage of the functionality that is there.  (e.g. it can use the browser to fetch web pages, interpret HTML, manage login cookies, etc.)
It also assumes it can interact with a user.  For example, setting up the default parser to handle a new site.

To work as a bot, all the functionality that is in the browser would need to be added to WebToEpub.  And if added, would complicate making it work in the browser.
Also, it would need changes to run non-interactively.

In short, it's a *lot* of work.  And I don't have enough interest to be willing to spend my limited free time on this.

Not to mention, A bot to do this work already exists.  (Novel-Grabber mentioned previously.)
So, I see no point in duplicating effort.  As there is an existing bot, it would be far more productive to spend time working on that.

@damare01, Given that there already is a telegram bot, why do you want another?
@gamebeaker Thanks for responding.

**Comment by damare01** [comment]
thank you for telling me your reason and I understand!
